README.md

```diff
@@ -154,15 +195,15 @@ Write Ruby code that will create an instance of a person.
 Your answer...
 
 ```ruby
-Replace this with your answer
+savraj = Person.new(name: "Savraj Singh", age: 24)
```

Comment:
:+1: although remember that `.new` doesn't automatically save something to the database. Either need to follow that up with `savraj.save` or start by doing `Person.create(name: "Savraj", age: 24)`